### PR TITLE
SPS's make combine sounds again

### PR DESCRIPTION
**Pull request by gurfan**
see title

:cl:
 * tweak: SPS units will make combine sounds again.

### CODE REVIEW

**Comment by jknpj** [comment]
```code/modules/telesci/gps.dm:273:warning: num: variable defined but not used```

File: code/game/machinery/computer/security_alert.dm [hunk 1]
```diff
@@ -97,7 +97,7 @@ TODO: literally every alarm but SPS alarms.
 	user << browse(listing, "window=security_alert")
 	onclose(user, "security_alert")
 
-/obj/machinery/computer/security_alerts/proc/receive_alert(var/alerttype, var/newdata, var/verbose = 1)
+/obj/machinery/computer/security_alerts/proc/receive_alert(var/alerttype, var/newdata, var/verbose)
```
Comment (SonixApache) [inline code/game/machinery/computer/security_alert.dm]: remove this change

**Comment by SonixApache** [comment]
This makes the SPS circumvent the SACs - with the current code, if all SACs are disabled, the SPS won't ever beep. That's what the `boop` var is for. You could remove your changes to that var if you'd like to keep that behavior.

I really don't see the point in making the SPS scream loudly (and only locally) whenever you strip/slay an officer, seems like a big uncessary buff to sec, but if that's what the people want, that's what we get.

File: code/modules/telesci/gps.dm [hunk 2]
```diff
@@ -258,10 +257,61 @@ var/list/SPS_list = list()
 	var/alertarea = get_area(SPS)
 	var/alerttime = worldtime2text()
 	var/verbose = TRUE
+	var/boop = TRUE
```
Comment (SonixApache) [inline code/modules/telesci/gps.dm]: remove this change too

File: code/modules/telesci/gps.dm [hunk 3]
```diff
@@ -258,10 +257,61 @@ var/list/SPS_list = list()
...
 		if(receiver && !receiver.stat)
 			receiver.receive_alert(alerttype, transmission_data, verbose)
 			boop = TRUE
-	if (boop)
-		playsound(src,'sound/machines/radioboop.ogg',40,1)
+	
+	if(boop)
+		deathsound(isdead)
+
+/obj/item/device/gps/secure/proc/deathsound(var/dead=FALSE)
```
Comment (SonixApache) [inline code/modules/telesci/gps.dm:270]: you should just cut all the refs to `dead` since you're only calling this when the wearer dies anyway

**Comment by drawsstuff** [comment]
merge when

**Pull request merged**